packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -0,0 +1,122 @@
+const Storage = window.localStorage;
+
+class CustomPropertyInspector {
+  constructor({propertiesElement, inspectorElement}) {
+    this.properties = this.setCustomProperties(propertiesElement);
+
+    this.renderInspector()
+  }
+
+  setCustomProperties(element) {
+    let prop
+    const styles = getComputedStyle(element)
+    const properties = {}
+
+    for (prop in styles) {
```

Comment:
```suggestion
    const styles = window.getComputedStyle(element)
    const properties = {}

    for (const [styleValue, styleName] of Object.entries(styles)) {
```

we should declare variables such that they are available only within the block scope of the loop where they are used. Going further, using `for...of` with an iterable that includes only the entries on the `CSSStyleDeclaration` object itself will save us from having to use conditional checks to exclude values from the prototype chain with `hasOwnProperty` below.

Reply:
hmmm playing with `CSSStyleDeclaration` a bit more - it seems to include numerical property keys that have string style name values that are present elsewhere in the same object as string property keys (the exception being vendor prefixed styles which don't contain prefixed hyphens). I can see why iterating with `for...of` may be problematic now